src/cmd/ksh93/meson.build

```diff
@@ -26,6 +26,7 @@ shared_c_args += [
     '-DSHOPT_COSHELL',
     '-DREAD_TIMEOUT=' + '@0@'.format(get_option('read-timeout')),
     '-DAUDIT_FILE=' + '"@0@"'.format(get_option('audit-file')),
+    '-DSHOPT_HISTEXPAND',
```

Comment:
Isn't this unnecessary with the above changes?

Reply:
Fixed.